pkg/secrets/secrets.go

```diff
@@ -127,15 +127,20 @@ func getMountsMap(path string) (string, string, error) {
 }
 
 // SecretMounts copies, adds, and mounts the secrets to the container root filesystem
-func SecretMounts(mountLabel, containerWorkingDir string, mountFile []string) []rspec.Mount {
-	var secretMounts []rspec.Mount
+func SecretMounts(mountLabel, containerWorkingDir string, mountFileFromFlag string) []rspec.Mount {
```

Comment:
nit: Just call it mountFile or overrideMountFile

Reply:
Won't that be confusing with the actual `OverrideMountsFile`?

Reply:
Agree with just mountFile

Reply:
Fixed.